src/measure_theory/function/l1_space.lean

```diff
@@ -552,6 +568,19 @@ begin
   simp [real.norm_eq_abs, ennreal.of_real_le_of_real, abs_le, abs_nonneg, le_abs_self],
 end
 
+lemma mem_ℒ1_to_real_of_lintegral_ne_top
+  {f : α → ℝ≥0∞} (hfm : ae_measurable f μ) (hfi : ∫⁻ x, f x ∂μ ≠ ∞) :
+  mem_ℒp (λ x, (f x).to_real) 1 μ :=
```

Comment:
@RemyDegenne is this what you had in mind or should I try to remove the `has_finite_integral` lemma also?

Reply:
It's fine if you keep a `has_finite_integral` lemma. I can change it later if needed (if indeed my refactor of `integrable` proves to be desirable).

To anybody else reading: I am thinking about redefining `integrable f μ` to mean `mem_ℒp f 1 μ`, hence I suggested that `integrable_to_real_of_lintegral_ne_top` could be proven by first proving a lemma about Lp then using `mem_ℒp_one_iff_integrable`.